packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -31,76 +32,105 @@ export class BulkApiImpl implements BulkApi {
     }
     /**
      * Bulk API: Get all partitions (nodes without parent) from the repo
-     * @param req no `parameters` or `body`
+     * @param request no `parameters` or `body`
      * @param response The list of all partition nodes, without children or annotations
      */
-    partitions = async (req: Request, response: Response): Promise<void> => {
-        logger.requestLog(` * partitions request received, with body of ${req.headers["content-length"]} bytes`)
-        const result = await this.ctx.bulkApiWorker.bulkPartitions()
-        lionwebResponse<PartitionsResponse>(response, result.status, result.queryResult)
+    listPartitions = async (request: Request, response: Response): Promise<void> => {
+        logger.requestLog(` * listPartitions request received, with body of ${request.headers["content-length"]} bytes`)
+        const clientId = getStringParam(request, "clientId")
+        if (isResponseMessage(clientId)) {
```

Comment:
Also here I do not understand the goal of `isResponseMessage`. Before it was checking the depthLimit and now the clientId